### ISSUE TITLE
Why recomment against using SDA anymore?

### ISSUE THREAD

**Issue by JameyDurst578** [Jessecar96/SteamDesktopAuthenticator#924]
In the recent versions, I saw that @Jessecar96 mentioned that using SDA reduces our security. I want to ask,  on which matters does this security problem occur? For example, does it mean that even when I take the necessary precautions to keep my data safe, will it be breached and stolen? Or is there any problem from Steam that prohibits or detects any SDA use and bans the account? I want to ensure the reasons of that security problems, so that I can consider taking that risk

**Comment by jwcloverain** [comment]
see readme
https://github.com/Jessecar96/SteamDesktopAuthenticator/blob/master/README.md
**FINALLY: Using this application is a bad idea, because it COMPLETELY DEFEATS THE PURPOSE of two-factor authentication! If your desktop is infected with a virus, it will be able to hijack the authenticator app and completely subvert the protection. THIS APPLICATION DOES NOT PROTECT YOUR ACCOUNT; IT ONLY ALLOWS YOU TO USE STEAM FEATURES THAT REQUIRE THE AUTHENTICATOR WITHOUT HAVING A PHONE. If you have a phone that supports the Mobile Authenticator, you really shouldn't use this application!**

**Comment by jwcloverain** [comment]
> > see readme https://github.com/Jessecar96/SteamDesktopAuthenticator/blob/master/README.md **FINALLY: Using this application is a bad idea, because it COMPLETELY DEFEATS THE PURPOSE of two-factor authentication! If your desktop is infected with a virus, it will be able to hijack the authenticator app and completely subvert the protection. THIS APPLICATION DOES NOT PROTECT YOUR ACCOUNT; IT ONLY ALLOWS YOU TO USE STEAM FEATURES THAT REQUIRE THE AUTHENTICATOR WITHOUT HAVING A PHONE. If you have a phone that supports the Mobile Authenticator, you really shouldn't use this application!**
> 
> This paragraph is included since the first version of SDA. I don't think the creator meant that when he rewrote the security warning again in the last version of the program. I am wondering is there any other security problems about this program instead of just getting computer virus etc. ?

Any program has unknown vulnerabilities including operating system, as stated in the readme You use this program at your own risk

**Comment by yurasiner** [comment]
But it is strange that he decided to add this inscription only now. And it is strange that the microsoft defender of windows began to warn about him only now. This was not the case in previous updates.

**Comment by LORD47** [comment]
> But it is strange that he decided to add this inscription only now. 

Are you sure about that? He always recommended against using SDA (same goes for any other 3rd party authinicator), for example the [ReadMe from 8 years ago](https://github.com/Jessecar96/SteamDesktopAuthenticator/blob/701a8728e227999c27d43c602dc637075caafb63/README.md) mentions that.


> And it is strange that the microsoft defender of windows began to warn about him only now. This was not the case in previous updates.

[VirusTotal scan ](https://www.virustotal.com/gui/file/1de48c7e9d8f5487587d447e3dbe69481b8716c5e8384b31f698664f14500e08?nocache=1)shows nothing malicious except one no-name AV, so can you elaborate more about this MS Defender warning?

**Comment by yurasiner** [comment]
> He always recommended against using SDA

Maybe somewhere when in readme. But if you look at https://github.com/Jessecar96/SteamDesktopAuthenticator/releases , then this inscription in the description appeared only on the last update



> [VirusTotal scan ](https://www.virustotal.com/gui/file/1de48c7e9d8f5487587d447e3dbe69481b8716c5e8384b31f698664f14500e08?nocache=1)shows nothing malicious except one no-name AV, so can you elaborate more about this MS Defender warning?

It's a smartscreen, it doesn't say anything specific, it just doesn't recommend running. I learned how smartscreen works, and it blocks such programs only if it finds it dangerous in the database

**Comment by AwkwordBG** [comment]
Is that thing normal though? 
https://www.virustotal.com/gui/file/dfc532c2ce2387f1077a24c85d4cbb83efd5085426289b14a976ac0322d9cefa 
I wasn't getting any warnings by Avast before the latest update. Now i get it every day at 11:30pm.
VirusTotal says there's Trojan, but the rar file is not infected. I don't get it.

**Comment by jessecor96** [comment]
Yes a lot of security issues have been fixed since the writing of that, however it is there to warn against the unexpected. If steam for some reason in the future decides that SteamAuth should only be accessed by mobile and for some reason decides to take  action, then the warning stands. We've already got Valves approval but you never know, I've got systems in place to warn if there is any risk. 

Version 13.1 (Fixes security breach): https://bit.ly/SDA-1-0-13-V

Hope this helps!
